### PR TITLE
NumTicketsInput enhancements

### LINKED ISSUE #2332: [1.5.0-rc2] Tickets tab: Weird alignment on "Not enough funds"
Screenshot from 2019-11-22 04-57-57

The number "1" and the orange "Not enough funds" text appear to be too high. Also I'm not sure the text needs to be wrapped onto two lines?

### PR DESCRIPTION
**Pull request by bgptr**
closes #2332

### CODE REVIEW

**Comment by bgptr** [comment]
Not my PR break it. I see this issue on the master too.
I'll fix it now quickly.

**Comment by vctt94** [comment]
oh, thanks! I haven't check master

**Comment by bgptr** [comment]
I've committed a quick fix.

File: app/components/views/TransactionsPage/SendTab/OutputRow.js [hunk 1]
```diff
@@ -57,42 +57,46 @@ const SendOutputRow = ({
...
-          required={true}
-          showErrors={error && error.amount}
-          invalid={error && error.amount}
-          invalidMessage={error && error.amount}
-          amount={value}
-          placeholder={intl.formatMessage(messages.amountPlaceholder)}
-          onChange={ e => onValidateAmount({ value: e.value , index, atomValue: e.atomValue })}
-          onKeyDown={onKeyDown}
-        />
+        <div class="send-input-wrapper"><AccountsSelect className="send-input"
```
Comment (vctt94) [inline app/components/views/TransactionsPage/SendTab/OutputRow.js]: className instead of class

File: app/style/SendPage.less [hunk 2]
```diff
@@ -159,12 +159,19 @@
...
+    margin-bottom: -14px;
+  }
+}
+
+.send-input {
+  width: 260px;  
   margin-left: 10px;
   margin-right: 10px;
-
+
```
Comment (vctt94) [inline app/style/SendPage.less]: white space

**Comment by bgptr** [comment]
I fixed them. Thanks

**Pull request merged**